### ISSUE TITLE
Filter events based on UserID

### ISSUE THREAD

**Issue by theevilbit** [redcanaryco/mac-monitor#3]
**At a high level -- can you summarize your request?**
It would be nice to filter events based on userID.

**What is the current alternative solution?**
Export all events and filter them.

**Are there "In-the-Wild" threats or corresponding ATT&CK techniques that exist for which this telemetry would be helpful?**
Not really, but the main idea is to monitor processes run as root to catch processes running elevated.

**Anything else?**
NA

**Comment by Brandon7CC** [comment]
I might reach out to you for a 👍 or 👎 on the UI changes when that time comes for this feature!

**Comment by Brandon7CC** [comment]
Thank you @theevilbit! Added in `v1.0.3`!
